src/gtest/test_sidechain_certificate_quality.cpp

```diff
@@ -771,7 +839,7 @@ TEST(SidechainsMultipleCerts, BlocksWithCertsOfSameOrIncreasingEpochsAndDecreasi
     cert_3.epochNumber = cert_1.epochNumber + 1;
     bBlock.vcert = {cert_1, cert_2, cert_3};
 
-    EXPECT_FALSE(CheckCertificatesOrdering(bBlock.vcert, dummyState));
+    EXPECT_TRUE(CheckCertificatesOrdering(bBlock.vcert, dummyState));   // This is now accepted for v2 sc
```

Comment:
Can we get rid of this comment?
```suggestion
    EXPECT_TRUE(CheckCertificatesOrdering(bBlock.vcert, dummyState));
```

Reply:
Yes we can.